src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/CreateClusterTests.cs

```diff
@@ -423,5 +423,75 @@ public void TestCreateLinuxSparkClusterWithComponentVersion()
                 Assert.Equal(result.State, AsyncOperationState.Succeeded);
             }
         }
+
+        [Fact]
+        public void TestCreateDefaultFsAzureBlobClusterUsingClusterParameters()
+        {
```

Comment:
@srsiva You need to mock CreateWhithHttpMessagesAsync - it is possible to mock some methods and not others by using an instance of the actual client in your Mock to make the actual API call, while using a mock response for other methods.